src/FixedSizeArrays.jl

```diff
@@ -94,6 +96,41 @@ end
 
 # helper functions
 
+dimension_count_of(::Base.SizeUnknown) = 1
+dimension_count_of(::Base.HasLength) = 1
```

Comment:
Last line still not covered (unless it's another coverage bug?)

Reply:
Yeah, it's JuliaLang/julia#54214 again. I guess this form is buggy:

```julia
f(::SingletonType) = isbits_literal
```

Reply:
So it isn't just empty tuple but any singleton?

Reply:
I guess :shrug:

Reply:
I definitely checked that these lines are in actuality covered by the tests, by replacing `1` with `error()`.